{"name": "تفشي_طبي_في_إيطاليا", "id": 7312579, "predecessors": [{"name": "الصحة_في_إيطاليا", "id": 940616}, {"name": "أمراض_واضطرابات_الإنسان", "id": 4257835}], "successors": [{"name": "جائحة_فيروس_كورونا_في_إيطاليا", "id": 7324806}]}